### ISSUE TITLE
Robolectric kills VM

### ISSUE THREAD

**Issue by serso** [robolectric/robolectric#597]
I got next weird exception while building app on the linux computer:

org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.15:test (default-test) on project calculatorpp-android-app: ExecutionException; nested exception is java.util.concurrent.ExecutionException: java.lang.RuntimeException: The forked VM terminated without saying properly goodbye. VM crash or System.exit called ?
Command was/bin/sh -c cd /home/serso/projects/java/android/calculatorpp/android-app && /usr/lib/jvm/java-6-sun-1.6.0.26/jre/bin/java -jar /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefirebooter2771085578585005504.jar /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefire6993464813379835914tmp /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefire_56898727128656163600tmp
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:213)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:84)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:59)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.singleThreadedBuild(LifecycleStarter.java:183)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:161)
    at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:320)
    at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:156)
    at org.apache.maven.cli.MavenCli.execute(MavenCli.java:537)
    at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:196)
    at org.apache.maven.cli.MavenCli.main(MavenCli.java:141)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
    at java.lang.reflect.Method.invoke(Method.java:597)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:290)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:230)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:409)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:352)
    at org.codehaus.classworlds.Launcher.main(Launcher.java:47)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
    at java.lang.reflect.Method.invoke(Method.java:597)
    at com.intellij.rt.execution.application.AppMain.main(AppMain.java:120)
Caused by: org.apache.maven.plugin.MojoFailureException: ExecutionException; nested exception is java.util.concurrent.ExecutionException: java.lang.RuntimeException: The forked VM terminated without saying properly goodbye. VM crash or System.exit called ?
Command was/bin/sh -c cd /home/serso/projects/java/android/calculatorpp/android-app && /usr/lib/jvm/java-6-sun-1.6.0.26/jre/bin/java -jar /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefirebooter2771085578585005504.jar /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefire6993464813379835914tmp /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefire_56898727128656163600tmp
    at org.apache.maven.plugin.surefire.SurefirePlugin.assertNoException(SurefirePlugin.java:177)
    at org.apache.maven.plugin.surefire.SurefirePlugin.handleSummary(SurefirePlugin.java:167)
    at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeAfterPreconditionsChecked(AbstractSurefireMojo.java:777)
    at org.apache.maven.plugin.surefire.AbstractSurefireMojo.execute(AbstractSurefireMojo.java:645)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:101)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:209)
    ... 25 more
Caused by: org.apache.maven.surefire.booter.SurefireBooterForkException: ExecutionException; nested exception is java.util.concurrent.ExecutionException: java.lang.RuntimeException: The forked VM terminated without saying properly goodbye. VM crash or System.exit called ?
Command was/bin/sh -c cd /home/serso/projects/java/android/calculatorpp/android-app && /usr/lib/jvm/java-6-sun-1.6.0.26/jre/bin/java -jar /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefirebooter2771085578585005504.jar /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefire6993464813379835914tmp /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefire_56898727128656163600tmp
    at org.apache.maven.plugin.surefire.booterclient.ForkStarter.runSuitesForkOnceMultiple(ForkStarter.java:252)
    at org.apache.maven.plugin.surefire.booterclient.ForkStarter.run(ForkStarter.java:165)
    at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeProvider(AbstractSurefireMojo.java:883)
    at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeAfterPreconditionsChecked(AbstractSurefireMojo.java:747)
    ... 28 more
Caused by: java.util.concurrent.ExecutionException: java.lang.RuntimeException: The forked VM terminated without saying properly goodbye. VM crash or System.exit called ?
Command was/bin/sh -c cd /home/serso/projects/java/android/calculatorpp/android-app && /usr/lib/jvm/java-6-sun-1.6.0.26/jre/bin/java -jar /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefirebooter2771085578585005504.jar /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefire6993464813379835914tmp /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefire_56898727128656163600tmp
    at java.util.concurrent.FutureTask$Sync.innerGet(FutureTask.java:222)
    at java.util.concurrent.FutureTask.get(FutureTask.java:83)
    at org.apache.maven.plugin.surefire.booterclient.ForkStarter.runSuitesForkOnceMultiple(ForkStarter.java:236)
    ... 31 more
Caused by: java.lang.RuntimeException: The forked VM terminated without saying properly goodbye. VM crash or System.exit called ?
Command was/bin/sh -c cd /home/serso/projects/java/android/calculatorpp/android-app && /usr/lib/jvm/java-6-sun-1.6.0.26/jre/bin/java -jar /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefirebooter2771085578585005504.jar /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefire6993464813379835914tmp /home/serso/projects/java/android/calculatorpp/android-app/target/surefire/surefire_56898727128656163600tmp
    at org.apache.maven.plugin.surefire.booterclient.ForkStarter.fork(ForkStarter.java:485)
    at org.apache.maven.plugin.surefire.booterclient.ForkStarter.fork(ForkStarter.java:352)
    at org.apache.maven.plugin.surefire.booterclient.ForkStarter.access$300(ForkStarter.java:85)
    at org.apache.maven.plugin.surefire.booterclient.ForkStarter$1.call(ForkStarter.java:224)
    at org.apache.maven.plugin.surefire.booterclient.ForkStarter$1.call(ForkStarter.java:214)
    at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
    at java.util.concurrent.FutureTask.run(FutureTask.java:138)
    at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
    at java.lang.Thread.run(Thread.java:662)

Basically, by some reason VM is killed (exited with System.exit()?). Also, I got similar behaviour when I run test from IDE - test exited with error code 10.

When I run tests on the Windows - everything is fine so far, may be problem in Java version?
